frontend/src/scenes/insights/filters/PathCleaningFilter.tsx

```diff
@@ -37,8 +37,8 @@ export function PathCleaningFilter(): JSX.Element {
             <Row align="middle" justify="space-between" style={{ paddingLeft: 0 }}>
                 <Popup
                     visible={open}
-                    placement={'bottom-end'}
-                    fallbackPlacements={['bottom-start']}
+                    placement={'top-end'}
+                    fallbackPlacements={['top-start']}
                     onClickOutside={() => setOpen(false)}
```

Comment:
This is a low down component so it was weird that we preferred bottom. Top makes more sense